tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "strictNullChecks": true,
+    "module": "none",
+    "target": "es6",
```

Comment:
Why es6? You should use es5 (to have better compatibility with old browsers

Reply:
done